pkg/api/speak-stream/v1/interfaces/interfaces.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 Deepgram SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+
+// This package defines interfaces for the live API
+package interfaces
+
+// SpeakMessageCallback is a callback used to receive notifications for platforms messages
+type SpeakMessageCallback interface {
+	Metadata(md *MetadataResponse) error
```

Comment:
I would maybe just have a comment dividing the WS Text vs Binary messages.

Something like:
```
# These are WS TextMessage that are used for flow control.
...

# These are WS BinaryMessage that are used to send audio data to the client
...
```

Reply:
@dvonthenen Do you mean like this:

```go
type SpeakMessageCallback interface {
        // These are WS TextMessage that are used for flow control.
	Metadata(md *MetadataResponse) error
	Flushed(fl *FlushResponse) error
	Error(er *ErrorResponse) error
	Close(cr *CloseResponse) error
	Open(or *OpenResponse) error
         // These are WS BinaryMessage that are used to send audio data to the client
	Binary(byMsg []byte) error 
}
```

Reply:
👍